src/main/java/uk/gov/hmcts/darts/cases/service/impl/CaseServiceImpl.java

```diff
@@ -131,14 +131,6 @@ public List<AdvancedSearchResult> advancedSearch(GetCasesSearchRequest request)
         return AdvancedSearchResponseMapper.mapResponse(hearings);
     }
 
-    public List<EventResponse> getEvents(Integer hearingId) {
-        Optional<HearingEntity> hearingEntity = hearingRepository.findById(hearingId);
-        if (hearingEntity.isEmpty()) {
-            throw new DartsApiException(CaseApiError.HEARING_NOT_FOUND);
```

Comment:
With this being removed, looks like the `HEARING_NOT_FOUND` enum no longer has usages, so can be removed as well from `CaseApiError`.